### DISCUSSION TITLE
Getting JSON response from Instagram stories API Python

### DISCUSSION THREAD

**Discussion by awaisdotcodes** [community/community#21431] [Other Feature Feedback, Questions, & Ideas]
<p>I got this <code>API</code> request when open a story on <code>Instagram</code> <a href="https://i.instagram.com/api/v1/feed/reels_media/?reel_ids=28465898840&amp;reel_ids=24898283344&amp;reel_ids=43075539583" class="inline-onebox" rel="noopener nofollow ugc">Login • Instagram</a><br>
But, the issue is that when you open the same link even in the browser, it gives this response back<br>
<code>{"message":"useragent mismatch","status":"fail"}</code><br>
How can we get the correct <code>response</code> back which is shown in the <code>network</code> tab? Or is it possible to get the content directly from the chrome dev tool network tab?</p>
<p>I have attached the screenshot of from <code>network tab</code> below<br>
<img src="https://user-images.githubusercontent.com/64072435/181064525-61d83fe4-a5ae-4155-a6a4-0b885583ae05.png" alt="image" data-base62-sha1="bVbabaUp6GoJWNRstZCYr6K64o4" width="433" height="93"></p>

**Comment by mpboom** [comment, marked as answer]
<p>Hey <a class="mention" href="/codewithawais">@CodeWithAwais</a> and welcome!</p>
<p>In general, it’s often not the most reliable to copy DevTools requests. Rather, try making use of the <a href="https://developers.facebook.com/docs/instagram-api/" rel="noopener nofollow ugc">official Instagram API</a>.</p>
<p>If you’re determined to follow this path, make sure everything in your Python session is the same as in your browser session. Think about not only URLs and their parameters, but also cookies and, what seems to break you up in this case, headers like the <code> User-Agent</code> header.</p>
<p>If you’re using the Python <code>requests</code> module, you can use a <code>request.Session()</code> and pass in those headers and cookies from the start.</p>

**Reply by gido3823** [reply]
![Screenshot_٢٠٢٢٠٧١٦-٠٣٠٠٥١_Instagram Lite](https://user-images.githubusercontent.com/88488087/186355639-b48c7bf1-1fc1-4ef5-86b4-d04900de790e.jpg)

**Reply by mpboom** [reply]
@gido3823 what is the question or comment? Please elaborate.
